src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -380,24 +391,25 @@ uct_ib_mlx5_ep_set_rdma_seg(struct mlx5_wqe_raddr_seg *raddr, uint64_t rdma_radd
 
 static UCS_F_ALWAYS_INLINE void
 uct_ib_mlx5_set_dgram_seg(struct mlx5_wqe_datagram_seg *seg, int is_global,
-                          struct mlx5_wqe_av *av, uint8_t path_bits)
+                          uct_ib_mlx5_base_av_t *av,
+                          struct mlx5_grh_av *grh_av, uint8_t path_bits)
```

Comment:
instead of is_global let's check (grh_av != NULL) this will allow to reduce ud_ep size in the future as well